modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,14 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for destroying firewalls in the host project
+[%collapsible]
+====
+* `compute.firewalls.delete`
+* `compute.firewalls.get`
+* `compute.firewalls.list`
+====
```

Comment:
The required permissions are `compute.firewalls.delete` and `compute.networks.updatePolicy` instead (see below for example).

FYI the custom role in the QE's host project for that purpose: 
```
$ gcloud iam roles describe --project openshift-qe-shared-vpc compute.firewalls.delete_compute.networks.updatePolicy
description: permissions of deleting firewall rules and update network policy
etag: BwYi1WE_uEs=
includedPermissions:
- compute.firewalls.delete
- compute.networks.updatePolicy
name: projects/openshift-qe-shared-vpc/roles/compute.firewalls.delete_compute.networks.updatePolicy
stage: ALPHA
title: compute.firewalls.delete_compute.networks.updatePolicy
$